components/Dialogs/src/MDCDialogPresentationController.h

```diff
@@ -16,6 +16,18 @@
 
 #import "MaterialShadowElevations.h"
 
+@class MDCDialogPresentationController;
+
+/**
+ Similar to UIPopoverPresentationControllerDelegate, MDCDialogPresentationControllerDelegate lets
+ you customize the behavior of a popover-based presentation.
```

Comment:
These docs don't sound correct (copy-pasted?). This API is unlikely to be similar to `UIPopoverPresentationControllerDelegate`; can you take another stab at these docs to be more aligned with the intended use case of the delegate?

Reply:
Copy pasted then modified slightly 🤙. I'll write more!